Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+size magnitudes. To do this the team will operate at several different inlet pressures and determine the correct size of these nozzles to create optimal expansion.
+The team will then plot this data and fit a polynomial approximation to it. This polynomial will serve as a initial guess for future nozzles that will be
+expanded upon and optimized during testing.
+
+% Below I have used subsections to identify key ideas in this section. These particular subsections are not required as part of the SPEX Standard, but serve as an example of using subsections in a text.
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project will include a lot of design and data analysis. Once the design phase
+is complete, testing the system will be the primary concern of the team. The project will start off with a slight redesign of the test stand, optmizing it for new conditions if necessary.
+Then comes the design of the nozzles. The team will need to effectively and efficiently determine the appropriate chamber and inlet pressures with varying
+nozzle dimensions to create the polynomial curve approximation that will be an integral part of the rest of the research on the topic. Once that is complete, the
+testing and experimentation phase begins. After the culmination of this phase, the team can try to design, fabricate, and test an aerospike nozzle, only if time permits.
+There are benefits to aerospikes but there are also disadvantages. The advantages are concerned with the region of optimal expansion. The aerospike nozzle can match the
+ambient pressure over a larger region and therefore can produce more thrust. However, the disadvantages are increased thermal loading, which is not terribly important here but is worht mentioning,
+and high design and fabrication complexity. This is the projected path for the investigation of the cold gas thruster.
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The deliverables of this project will be split into to two sections, physical and non-phsyical.
+
+\subsubsection{Physical Deliverables}
+\label{subsubsec: physical deliverables}
+
+The physical deliverables of this project with the CGT\@. This includes the propulsion system and the test stand. The propulsion system will include a regulator,
+a solenoid, a tank, a nozzle, piping, and the interfaces all connected in a series. The test stand will include the t-slot frame with the tank frame. They will be made entirely of aluminum. The interfaces between the aluminum parts will also be included. This
+consists of the various t-slot connections and the bolts and washers that connect the aluminum sheets.
+
```

Comment:
This is the best I can do. I don't know much about how arduinos are used so this is a guess. I can't get David to download and install LaTeX. 

`There will also be an arduino connected to a computer that will use Simulink for data acquisition. The arduino will be a conduit to relay the data from the load cell
to Simulink. There will also be an arduino to control the use of the solenoid. The solenoid will be activated from the Simulink program, which will coincide with 
the beginning of data acquisition. `